### DISCUSSION TITLE
HACS install - Config flow could not be loaded: undefined

### DISCUSSION THREAD

**Discussion by wwj1978** [leikoilja/ha-google-home#491] [Q&A] [answered]
When I try to add integration I get this message: `Config flow could not be loaded: undefined`. This is very basic installation oa HA on Rpi3

**Comment by leikoilja** [comment, marked as answer]
@wwj1978, please use issues tab https://github.com/leikoilja/ha-google-home/issues to report the bug. But with that fill in all the requested information. Without it and your logs we won't be able to help you.
